td-layout/src/build_time.rs

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2021 Intel Corporation
+//
+// SPDX-License-Identifier: BSD-2-Clause-Patent
+
```

Comment:
Isn't this file supposed to be generated by `td-layout-config`? 

It is missing the
```
// Auto-generated by `td-layout-config`, do not edit manually.
```

Reply:
The above comment is added after generating runtime/build.rs. Will regenerate them.